compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/KnownTargets.cpp

```diff
@@ -116,9 +117,12 @@ TargetAttr createTargetAttr(const TargetDetails &details, StringRef arch,
       DictionaryAttr{});
 
   TargetChipAttr targetChip;
-  if (details.chip)
-    targetChip =
-        TargetChipAttr::get(context, details.chip->wgpCount, DictionaryAttr{});
+  if (details.chip) {
+    StringAttr skuAttr =
+        StringAttr::get(context, details.chip->sku.value_or(""));
+    targetChip = TargetChipAttr::get(context, details.chip->wgpCount, skuAttr,
```

Comment:
If sku is optional, why are we setting it with an empty string? Is empty string considered the same as `nullptr` in `StringAttr`?